draft-ietf-rats-corim.md

```diff
@@ -769,68 +769,79 @@ The types defined for a group identified are UUID and variable-length opaque byt
 {::include cddl/group-id-type-choice.cddl}
 ~~~
 
-##### Measurements
+##### Claims
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A Claim is a key, value pair that in CoRIM represents a single piece of actual state or a reference state.
```

Comment:
["key-value pair"](https://en.wikipedia.org/wiki/Name–value_pair)

Reply:
If we're following the endorsements draft terminology it would read something like: "A Claim is a key-value pair that in CoRIM represents a single piece of actual or possible state.